4_visualize.R

```diff
@@ -0,0 +1,44 @@
+source('4_visualize/src/plot_results.R')
+
+p4 <- list(
+  # Plots for each lake-gcm combo
+  tar_target(
+    p4_20yr_gcm_preds_ice,
+    plot_20yr_gcm_preds_ice(p2_glm_uncalibrated_run_groups, 
+                    outfile_template <- '4_visualize/out/Site_20yr_preds_ice_%s_%s.png'),
+    pattern = map(p2_glm_uncalibrated_run_groups),
+    format='file'),
+  
+  # Plots for each lake
+  tar_target(
+    p4_20yr_average_preds_ice,
+    plot_20yr_average_preds_ice(p2_glm_uncalibrated_lake_groups,
+                                outfile_template <- '4_Visualize/out/Site_20yr_preds_ice_%s.png'),
+    pattern = map(p2_glm_uncalibrated_lake_groups),
+    format = 'file'
+  ),
+  
+  tar_target(
+    p4_20yr_average_profiles,
+    plot_20yr_average_profiles(p2_glm_uncalibrated_lake_groups,
```

Comment:
these depth profiles as averages is pretty neat! ([link to the image](https://user-images.githubusercontent.com/54007288/160445279-14d2d22e-519c-420c-bf61-a1f0b9b512b2.png) you shared in the main PR description for context if someone else sees this)